index.html

```diff
@@ -30,17 +31,18 @@
           </ul>
         </div>
 
-        <nav class="nav-link">
-          <div class="nav-menu">
-            <div class="ambuger-menu">
-              <span class="bar"></span>
-              <span class="bar"></span>
-              <span class="bar"></span>
+          <nav class="nav-link">
+            
+            <div class="nav-menu">
+              
+              <div class="ambuger-menu">
+                
+              </div>
             </div>
-          </div>
-        </nav>
-      </div>
-    </header>
+          </nav>
```

Comment:
- [ ] [REQUIRED] Good job so far! Your `Hamburger` button is not working and was not created because it displays nothing. Please implement my suggestions below:

* On mobile the hamburger menu button is visible.
* When the user clicks (or taps) the hamburger button, the mobile menu appears.
* When the user clicks (or taps) the close (`X`) button, the mobile menu `disappear`.
* When the user clicks (or taps) any of the mobile menu options, the correct part of the page is displayed.
* When the user clicks (or taps) any option of the mobile menu, the mobile menu `disappear`.

![image](https://user-images.githubusercontent.com/63915024/210359349-df3b1775-c8f0-432c-8656-ed5d015c31e4.png)